doc-content/enterprise-only/springboot/openshift-springboot-proc.adoc

```diff
@@ -0,0 +1,123 @@
+[id='openshift-springboot-proc_{context}']
+= Running a SpringBoot business application on {OPENSHIFT}
+
+To run your {PRODUCT} SpringBoot business application on {OPENSHIFT}, create an immutable image and push this image to your {OPENSHIFT} environment.
+
+ifdef::PAM[]
+Optionally, you can also use {CENTRAL} Monitoring to monitor the execution of business processes in your application.
+endif::PAM[]
+
+.Prerequisites
+
+* You developed a {PRODUCT} SpringBoot business application. For instructions about creating the application, see xref:bus-app-create_{context}[].
+* If necessary, you configured Spring security for the application. For instructions about configuring Spring security, see xref:bus-app-security-con_{context}[].
+* You completed other Spring configuration for the business application, as necessary. For instructions about Spring configuration for your business application, see xref:bus-app-configure-con_{context}[].
+* You created a single JAR file for the business application. For instructions about creating a single JAR file for your SpringBoot business application, see xref:creating-self-contained-image-proc_{context}[].
+* You are logged on to your {OPENSHIFT} environment using the `oc` command and the required project is active.
+ifdef::PAM[]
+* If you want to use {CENTRAL} Monitoring, you installed {CENTRAL} Monitoring using the operator. For instructions about installing {CENTRAL} Monitoring and other {PRODUCT} components using the operator, see {URL_DEPLOYING_ON_OPENSHIFT}#assembly-openshift-operator[_{DEPLOYING_OPENSHIFT_OPERATOR}_]. For convenience, use the same project/namespace to deploy {CENTRAL} Monitoring and your business application.
```

Comment:
Suggested rewrite:

You can use the same project/namespace to deploy {CENTRAL} Monitoring and your business application.

Remove the "/" and rewrite the text using and,of, or or. I suggest, "the same project or namespace"